charts/kyverno/templates/hooks/clusterrole.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "kyverno.pre-upgrade-job.name" . }}
+  annotations:
+    "helm.sh/hook": pre-upgrade
+    "helm.sh/hook-weight": "-10"
+rules:
+- apiGroups: [""]
+  resources: [configmaps]
+  verbs: [get, list, patch, create, delete]
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: [customresourcedefinitions]
+  verbs: [get, list, patch, create, delete, watch]
+- apiGroups: ["apps"]
+  resources: [deployments]
+  verbs: [get, list, patch, create, delete]
+- apiGroups: [""]
+  resources: [pods]
+  verbs: [list, get, delete,create]
```

Comment:
I'm not seeing a need to be able to `delete` or `create` pods so remove those capabilities.  I just see the need to `list`/`get` them.

Reply:
Implemented review comment.